### PR TITLE
MC-40582: Update the install page with how to download the SUT tool via command line

### PR DESCRIPTION
**Pull request by danidelcar**
## Purpose of this pull request

This pull request (PR) updates the install page from the Safe Upgrade Tool (SUT) topics with how to download the SUT tool via command line.

## Affected DevDocs pages

This PR modifies the following pages:

- [SUT welcome page](https://devdocs.magento.com/safe-upgrade-tool/introduction.html)
- [Prerequisites for SUT](https://devdocs.magento.com/safe-upgrade-tool/prerequisites.html)
- [SUT installation page](https://devdocs.magento.com/safe-upgrade-tool/install.html)
- [SUT use page](https://devdocs.magento.com/safe-upgrade-tool/run.html)
- Page-header in devdocs project for the SUT ALPHA release.

## Other information

JIRA [MC-40582](https://jira.corp.magento.com/browse/MC-40582)

whats new
Updated the [Install Safe Upgrade Tool](https://devdocs.magento.com/safe-upgrade-tool/install.html) topic with information on how to download the Safe Upgrade Tool via command line.

### CODE REVIEW

File: src/_includes/layout/page-header.html [hunk 1]
```diff
@@ -44,7 +44,7 @@ <h2 class="page-subtitle no_toc">{{ page.subtitle }}</h2>
 
   {% if page.url contains "safe-upgrade-tool/" %}
   <div class="message-banner">
-    This tool is still in ALPHA with limited scope. If you are a Magento Commerce user you can download it at <a href="https://repo.magento.com/">Magento repo</a>.
+    This tool is an ALPHA version with limited scope, available for all Magento Commerce merchants, only validating PHP Magento APIs and GraphQL schema. If you are a Magento Commerce user you can download it at the <a href="https://repo.magento.com/">Magento repo</a>.
```
Comment (dobooth) [inline src/_includes/layout/page-header.html:47]:
```suggestion
    This tool is an ALPHA version with limited scope, available for all Magento Commerce merchants, and only validating PHP Magento APIs and GraphQL schema. If you are a Magento Commerce user you can download it at the <a href="https://repo.magento.com/">Magento repo</a>.
```

**Review by dobooth — APPROVED** [review]

File: src/_includes/layout/page-header.html (hunk 1 above)
Comment (danidelcar) [reply src/_includes/layout/page-header.html:47]: thanks, corrected it!

File: src/safe-upgrade-tool/run.md [hunk 2]
```diff
@@ -24,10 +21,16 @@ bin/sut upgrade:check INSTALLATION_DIR
...
-
 The `INSTALLATION_DIR` value is the directory where your Magento instance is located.
 
+We recommend running the following command to avoid memory limitations:
+
+```bash
+php -d memory_limit=-1 /bin/sut
+```
+
+As well, it is recommended to use the `-m` command to run the tool against a specific module.
```
Comment (leslietilling) [inline src/safe-upgrade-tool/run.md:32]: Just a minor editorial suggestion
```suggestion
We also recommend using the `-m` command to run the tool against a specific module.
```

**Review by leslietilling — APPROVED** [review]
A minor suggested edit and this should be good to go :)

File: src/safe-upgrade-tool/run.md (hunk 2 above)
Comment (danidelcar) [reply src/safe-upgrade-tool/run.md:32]: done

**Review by jcuerdo — APPROVED** [review]

**Comment by danidelcar** [comment]
running tests

**Comment by danidelcar** [comment]
running tests

**Pull request merged**